orchestrator-salt/src/main/resources/salt/salt/fluent/template/td-agent.conf.j2

```diff
@@ -184,6 +184,22 @@
   tag {{fluent.providerPrefix}}.HDFS-SECONDARYNAMENODE
   read_from_head true
 </source>
+<source>
+  @type tail
+  format none
+  path /var/log/knox/gateway.log*
```

Comment:
Not sure you were going to cover it elsewhere, but we should also have monitoring for IDBroker logs.

Reply:
@lajosrodek , is it be on the machine where you have knox gateway? (can be interesting, maybe we won't use the same instance profile for idbroker machines, that can be problematic for s3 log support).. do you know the path of those logs? then i can include those as well

Reply:
No, IDBroker is always deployed to a separate instance (or instances, in case of HA). See `"refName": "idbroker"` in the [non-HA](https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/defaults/blueprints/cdp-sdx.bp) and [medium-HA](https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha.bp) BPs.

The IDBroker identity (EC2 instance profile or MSI) will always contain the baseline one, so whatever grants are needed for fluentd logging etc. will be also applied to IDBroker nodes: see [this doc section](https://docs.google.com/document/d/1DDUpWOIaKq-lbmTEyUoYnK8NIfrlRrGN54DWIms4S4E/edit?ts=5cef1597#heading=h.st7ehb8q9pnm).

IDBroker logs are found here:
```
[root@ip-10-65-64-185 ~]# ls -l /var/log/knox/idbroker/
total 64
-rw-r--r-- 1 knox knox  1480 Jul 18 12:45 gateway-audit.log
-rw-r--r-- 1 knox knox 41028 Jul 18 12:45 gateway.log
-rw-r--r-- 1 knox knox 16087 Jul 18 12:26 knoxcli.log
drwxr-xr-x 2 knox knox     6 Jul 18 12:26 stacks
[root@ip-10-65-64-185 ~]# ls -l /var/log/knox/idbroker/stacks/
total 0
```

Reply:
ok, thanks. overall it won't hurt if i include these as well